src/pages/en/guides/content-collections.md

```diff
@@ -165,6 +165,28 @@ You can use all of Zod’s properties and methods with content schemas. This inc
 
 📚 See [Zod’s documentation](https://github.com/colinhacks/zod) for a complete list of features.
 
+### Custom entry slugs
```

Comment:
```suggestion
## Custom entry slugs
```
Suggesting this as a separate entry in `defineCollections` and not a part of defining a schema